pkg/services/blockfetcher/blockfetcher.go

```diff
@@ -34,9 +35,15 @@ const (
 	// defaultTimeout is the default timeout for NeoFS requests.
 	defaultTimeout = 5 * time.Minute
 	// defaultOIDBatchSize is the default number of OIDs to search and fetch at once.
-	defaultOIDBatchSize = 8000
+	defaultOIDBatchSize = bqueue.DefaultCacheSize
 	// defaultDownloaderWorkersCount is the default number of workers downloading blocks.
 	defaultDownloaderWorkersCount = 100
+	// defaultIndexFileSize is the default size of the index file.
+	defaultIndexFileSize = 128000
+	// defaultBlockAttribute is the default attribute name for block objects.
+	defaultBlockAttribute = "Block"
+	// defaultIndexFileAttribute is the default attribute name for index file objects.
+	defaultIndexFileAttribute = "Index"
```

Comment:
Make a separate package for NeoFS blockstorage related code, move these common constants to this package, reuse these constants as defaults not only for BlockFetcher config, but also for uploader.